frontend/components/pricing/PremiumAddonBox.vue

```diff
@@ -9,509 +9,351 @@ import {
+})
+
+const totalYearlyPricePerMonth = computed(() => {
+  return (addon.price_per_year_eur / 12) * purchaseQuantity.value
+})
+
+const totalYearlyPrice = computed(() => {
+  return addon.price_per_year_eur * purchaseQuantity.value
+})
+
+// we don't charge per Validator anymore, but this is used to show users that the price
+// they used to pay per Validator hasn't changed now that we charge by Effective Balance
+const pricePerValidator = computed(() => {
+  const maxValidatorEffectiveBalancePrePectra = Number(formatValue(isNetworkGnosis ? '1' : '32', {
+    from: 'base', to: 'gwei',
+  }))
+
+  return (pricePerUnit.value * maxValidatorEffectiveBalancePrePectra)
+    / addon.extra_dashboard_effective_balance
+})
+
+const yearlySubscriptionSavings = computed(() => {
+  return (addon.price_per_month_eur * 12 - addon.price_per_year_eur)
+    * purchaseQuantity.value
 })
 
+const extraEffectiveBalance = computed(() =>
+  formatAmount(`${addon.extra_dashboard_effective_balance}`, {
+    minimumFractionDigits: 0,
+    sourceUnit: 'gwei',
+    targetCurrency: displayCurrencyDefault.main,
+
+  }),
+)
+
 const addonSubscriptionCount = computed(() => {
   return (
     user.value?.subscriptions?.filter(
       sub =>
         sub.product_category === ProductCategoryPremiumAddon
-        && (sub.product_id === props.addon.product_id_monthly
-          || sub.product_id === props.addon.product_id_yearly),
+        && (sub.product_id === addon.product_id_monthly
+          || sub.product_id === addon.product_id_yearly),
     ).length || 0
   )
 })
 
-const addonButton = computed(() => {
-  let text = $t('pricing.get_started')
-  if (isLoggedIn.value) {
-    text
-      = addonSubscriptionCount.value > 0
-        ? $t('pricing.addons.button.manage_addon')
-        : $t('pricing.addons.button.select_addon')
-  }
-
-  async function callback() {
-    if (isStripeDisabled.value) {
-      return
-    }
-
-    if (isLoggedIn.value) {
-      if (addonSubscriptionCount.value > 0) {
-        await stripeCustomerPortal()
-      }
-      else {
-        await stripePurchase(
-          props.isYearly
-            ? props.addon.stripe_price_id_yearly
-            : props.addon.stripe_price_id_monthly,
-          quantityForPurchase.value,
-        )
-      }
-    }
-    else {
-      await navigateTo({
-        path: '/login', query: { promoCode },
-      })
-    }
-  }
-
-  return {
-    callback,
-    disabled: isStripeDisabled.value,
-    text,
-  }
+const isQuantityLimitReached = computed(() => {
+  return purchaseQuantity.value >= maximumPurchaseQuantityAllowed.value
 })
 
-const maximumQuantity = computed(() => {
+const maximumPurchaseQuantityAllowed = computed(() => {
   return Math.floor(
-    ((props.maximumValidatorLimit || 10000)
-      - (user.value?.premium_perks.validators_per_dashboard || 0))
-    / props.addon.extra_dashboard_validators,
+    (effectiveBalancePerDashboardLimit
+      - (user.value?.premium_perks.effective_balance_per_dashboard || 0))
```

Comment:
issue: `premium_perks` is exposed by `useUserStore()`
```suggestion
      - (premium_perks.value?.effective_balance_per_dashboard || 0))
```

Reply:
changed